react-facebook-login/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for react-facebook-login 3.4
+// Project: https://github.com/keppelen/react-facebook-login
+// Definitions by: Alexandre Paré <https://github.com/apare>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react";
+
+export interface ReactFacebookLoginProps {
+    appId: string;
+    callback: (userInfo: ReactFacebookLoginInfo) => void;
+
+    autoLoad?: boolean;
+    buttonStyle?: React.CSSProperties;
+    containerStyle?: React.CSSProperties;
+    cookie?: boolean;
+    cssClass?: string;
+    disableMobileRedirect?: boolean;
+    fields?: string;
+    icon?: string | React.ReactNode;
+    isDisabled?: boolean;
+    language?: string;
+    onClick?: () => void;
+    reAuthenticate?: boolean;
+    redirectUri?: string;
+    scope?: string;
+    size?: "small" | "medium" | "metro";
+    textButton?: string;
+    typeButton?: string;
+    version?: string;
+    xfbml?: boolean;
+}
+
+export interface ReactFacebookLoginInfo {
+    id: string;
+    name: string;
+}
+
+export interface ReactFacebookLoginState {
+    isSdkLoaded?: boolean;
+    isProcessing?: boolean;
+}
+
+export default class ReactFacebookLogin extends React.Component<ReactFacebookLoginProps, ReactFacebookLoginState> {
```

Comment:
The comment here was to remove the `export default` from the declrarion file, and use `--allowSyntheticDefaultImports` in your project where you are importing `react-facebook-login`. 

the declaration file should end with `export = ReactFacebookLogin ` instead of `export default`